File: api_checker/app/src/main/java/io/kokoichi/sample/api_checker/MainActivity.java
package io.kokoichi.sample.api_checker;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    //レイアウト要素
    Button button;
    TextView titleView;
    TextView dateView;
    TextView descriptionView;

    //API
    private final String API_URL_PREFIX = "www.googleapis.com";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        button = findViewById(R.id.button);
        titleView = findViewById(R.id.titleView);
        dateView = findViewById(R.id.dateView);
        descriptionView = findViewById(R.id.descriptionView);

        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
//                MyAsync async = new MyAsync(titleView, dateView, descriptionView);
                FaceDetect async = new FaceDetect(titleView, dateView, descriptionView, getResources());
                async.execute();
            }
        });
    }
    // ここにAPI通信処理を書きます。
}

File: FirstApp/app/src/main/java/jp/mydns/kokoichi0206/firstapp/MainActivity.kt
package jp.mydns.kokoichi0206.firstapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Button
import android.widget.TextView
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btn = findViewById<Button>(R.id.myButton)
        btn.setOnClickListener {
            Log.d("hoge", "btn clicked")

            val textView = findViewById<TextView>(R.id.textView)
            textView.text = "Yeeeeeeeaaahhh"

            Toast.makeText(this, "Toast Toast !", Toast.LENGTH_LONG).show()
        }
    }
}